.gitignore

```diff
@@ -23,6 +23,9 @@ out
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
+# Project-specific stuff:
+.idea/*
```

Comment:
I don't know if this would cause anyone trouble, if so I will of course remove it. I just added it because git wanted to commit the "javaparser.iml" file from the IntelliJ IDEA project.

Reply:
Probably had to do with how I imported the project, did it another way now and it seems to be fine now. I reverted the commit.

Reply:
Thanks!